### PR TITLE
[SYS-001] adding first feature ocb build

### PR DESCRIPTION
**Pull request by d-nagaraj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# test_github_tutorial
\ No newline at end of file
+# test_github_tutorial
+## Workflow for github usage
```
Comment (jeevankumarr) [inline README.md]: Please add a description here.

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
-# test_github_tutorial
```
Comment (jeevankumarr) [inline README.md:1]: This neednt be removed

File: README.md (hunk 1 above)
Comment (d-nagaraj) [reply README.md]: ok, will work on it

File: README.md (hunk 2 above)
Comment (jeevankumarr) [inline README.md:1]: These changes need a more detailed review.

Comment (d-nagaraj) [reply README.md:1]: ok i will work on it

File: README.md [hunk 3]
```diff
@@ -1 +1,2 @@
...
\ No newline at end of file
+# test_github_tutorial
```
Comment (jeevankumarr) [inline README.md]: Need better casing for better readability

**Review by jeevankumarr — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (d-nagaraj) [reply README.md]: ok

File: README.md (hunk 2 above)
Comment (d-nagaraj) [reply README.md:1]: ok new patch set to be uploaded

**Review by jeevankumarr — APPROVED** [review]

**Pull request merged**